soh/soh/SaveManager.cpp

```diff
@@ -57,7 +57,13 @@ void SaveManager::LoadRandomizerVersion1() {
     if(!CVar_GetS32("gRandomizer", 0)) return;
 
     for (int i = 0; i < ARRAY_COUNT(gSaveContext.itemLocations); i++) {
-        SaveManager::Instance->LoadData("get" + std::to_string(i), gSaveContext.itemLocations[i].get);
```

Comment:
This is going to change a bit with #1421

Reply:
Yup I'm prepared to handle that merge if necessary.

Reply:
Merge handled